code/game/objects/items/storage/book.dm

```diff
@@ -225,6 +224,19 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 				EX.icon_state = "ghost1"
 				EX.name = "Purified [initial(EX.name)]"
 			user.visible_message("<span class='notice'>[user] has purified [SS]!</span>")
+	else if(istype(A, /obj/item/nullrod/scythe/talking))
```

Comment:
i would generalize this to work with any object that has a shade in it like soul stones, since what this block of code actually does is remove a shade from the sword

Reply:
aye, but how would I check that an object would have a shade in it and thus could be exorcised?

Reply:
Off the top of my head you do something like
for(mob/living/shade/varname in A.contents)
varname.remove()

Reply:
what I mean is, wouldn't that cause clicking on anything that a bible doesn't have a special interaction with to make that loud sound and make an exorcism attempt against that object?

Reply:
..no? If it doesn't find a shade you just wouldn't run that code

Reply:
you would have to differentiate between cleansing (soulstone) and exorcising (talking sword)